src/Core/Commands/DeleteOrganizationUserCommand.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Bit.Core.Commands.Interfaces;
+using Bit.Core.Exceptions;
+using Bit.Core.Repositories;
+using Bit.Core.Services;
+
+namespace Bit.Core.Commands;
+
+public class DeleteOrganizationUserCommand : IDeleteOrganizationUserCommand
```

Comment:
Please organize this command under `Core/OrganizationFeatures/OrganizationUsers`

Reply:
and following on from this logic, we should group the commands and queries in SCIM under

`Scim/Users/` and `Scim/Groups/`.

Your call whether it's still useful to have sub-folders for queries vs. commands.

Reply:
Thanks for clearing that up, I didn't know where it should exactly go.
I don't think theres a need for the extra folder, I've moved `GetUserQuery` to `Scim/Users/`.